### PR TITLE
ALL option in genre selection

### PR DESCRIPTION
**Pull request by eladrb**
ALL option in genre selection

### CODE REVIEW

File: public/js/movies.js [hunk 1]
```diff
@@ -75,15 +76,37 @@ function onSelectMovieSearch(e) {
   });
 }
 
-
-
 function onSelectGenreSearch(event) {
   const genre = event.target.value;
+  console.log(genre)
```
Comment (galcohenh) [inline public/js/movies.js:81]: remove the console log

File: public/js/movies.js [hunk 2]
```diff
@@ -75,15 +76,37 @@ function onSelectMovieSearch(e) {
...
+      },
+    });
+  }
+  else{
   $.ajax({
     url: `/api/movies/genre/${genre}?t=${new Date().getTime()}`,
     type: "GET",
     dataType: "json",
     success: function (response) {  
+      console.log(response);
```
Comment (galcohenh) [inline public/js/movies.js:109]: remove the console log

File: services/movie.js [hunk 3]
```diff
@@ -127,5 +125,5 @@ module.exports = {
   deleteMovie,
   getMovieByScreenId,
   groupMoviesByGenre,
-  editMovie
-};
+  editMovie,
```
Comment (galcohenh) [inline services/movie.js:128]: remove the comma

File: views/movies.ejs [hunk 4]
```diff
@@ -52,6 +52,7 @@
       <option value="Family">Family</option>
       <option value="Action">Action</option>
       <option value="Sci-fi">Sci-fi</option>
+      <option value="ALL">ALL</option>
```
Comment (galcohenh) [inline views/movies.ejs:55]: all should be the first option

File: public/js/movies.js (hunk 1 above)
Comment (danielmargolin) [reply public/js/movies.js]: remove console log

**Pull request merged**